pinot-core/src/main/java/org/apache/pinot/core/operator/query/FilteredGroupByOperator.java

```diff
@@ -60,8 +61,10 @@ public class FilteredGroupByOperator extends BaseOperator<GroupByResultsBlock> {
   private long _numEntriesScannedPostFilter;
   private final DataSchema _dataSchema;
   private final QueryContext _queryContext;
+  private final IdentityHashMap<AggregationFunction, Integer> _resultHolderIndexMap;
```

Comment:
We probably can keep it as local variable. Don't see it being used in other places

Reply:
For this operator, `getNextBlock` can only be called once, so it should be fine. For the same reason, `FilteredAggregationOperator` is also keeping it local